src/spectral/utils/spectral-validator.js

```diff
@@ -28,16 +28,16 @@ const parseResults = function(results, debug) {
         if (typeof severity === 'number' && code && message && path) {
           if (severity === 0) {
             // error
-            messages.addMessage(path, message, 'error');
+            messages.addMessage(path, message, 'error', code);
```

Comment:
I admit I don't know what the values for `code` look like but if it's not obvious that they are Spectral rules, it may lead to confusion as to where that "key" is in the ruleset. Perhaps we should add a distinction to this string and document that Spectral rules are configured a different way, so that users can figure out what to do when they see a Spectral rule.

Reply:
I'm hesitant to change the code displayed since that weakens the connection to the actual Spectral rule, but I sympathize with your concern so I've added some text to the README to hopefully make this clear.